util/activity-report/monthly-txs.js

```diff
@@ -0,0 +1,40 @@
+const format = require('pg-format');
+const db = require('../../dbPool.js');
+const { getChainTableName, validateChain } = require('./validate-chain.js');
+
+async function queryDb(chain) {
+  const queryStr = `
+    SELECT COUNT(*) AS monthly_txs
+    FROM %I.block_transactions t
+    WHERE t.signed_at 
+    BETWEEN DATE_TRUNC('month', NOW()) - INTERVAL '1 month' 
+    AND DATE_TRUNC('month', NOW())
```

Comment:
I think we need to clarify the requirements here. this particular `DATE_TRUNC` will always get the last calendar month. but maybe what we need is today minus 1 month.

e.g. if query is done on 5th Dec, do we get 1st Nov to 1st Dec ; or do we get 5th Nov to 5th Dec?

if we want the latter, perhaps `DATE_TRUNC` should be switched to `day` instead.